README.md

```diff
@@ -16,6 +16,7 @@ cmake  --build --preset ninja-multi-vcpkg
 ## Install & Integrate
 
 Simply use cmake install infrastructure:
+NOTE: if using the vcpkg preset, `VCPKG_ROOT` must be set to the root directory of the vcpkg instance
```

Comment:
This should go in the section above

Reply:
oops, good catch https://github.com/lifting-bits/gap/pull/43/commits/4804e64e062c9b9d1d528fc09be1bea356213b82